tests/fixtures/bug_features/whiteboard.json

```diff
@@ -1,2 +1,9 @@
VED", "cf_crash_signature": "", "is_open": false, "severity": "major", "cf_tracking_firefox_esr60": "---", "flags": [], "cf_status_firefox64": "---", "component": "Graphics", "cf_status_geckoview64": "---", "target_milestone": "---", "product": "Core", "cf_platform_rel": "---", "history": [{"when": "2017-07-04T02:36:27Z", "who": "yfdyh000@gmail.com", "changes": [{"removed": "UNCONFIRMED", "added": "RESOLVED", "field_name": "status"}, {"removed": "Unspecified", "added": "x86", "field_name": "platform"}, {"removed": "Untriaged", "added": "Graphics", "field_name": "component"}, {"removed": "", "added": "yfdyh000@gmail.com", "field_name": "cc"}, {"removed": "", "added": "DUPLICATE", "field_name": "resolution"}, {"removed": "Firefox", "added": "Core", "field_name": "product"}, {"removed": "normal", "added": "major", "field_name": "severity"}, {"removed": "Unspecified", "added": "Windows 8", "field_name": "op_sys"}, {"removed": "", "added": "2017-07-04T02:36:27Z", "field_name": "cf_last_resolved"}]}, {"when": "2017-07-04T05:00:38Z", "who": "dvander@alliedmods.net", "changes": [{"removed": "", "added": "dvander@alliedmods.net", "field_name": "cc"}, {"removed": "", "added": "needinfo?(johngraciliano@gmail.com)", "field_name": "flagtypes.name"}]}, {"when": "2017-07-04T07:53:38Z", "who": "Virtual@teknik.io", "changes": [{"removed": "", "added": "Virtual@MsgSafe.io", "field_name": "cc"}]}, {"when": "2017-07-17T21:46:57Z", "who": "dvander@alliedmods.net", "changes": [{"removed": "needinfo?(johngraciliano@gmail.com)", "added": "", "field_name": "flagtypes.name"}]}], "comments": [{"text": "User Agent: Mozilla/5.0 (Windows NT 6.2; Win64; x64; rv:56.0) Gecko/20100101 Firefox/56.0\nBuild ID: 20170701030203\n\nSteps to reproduce:\n\nUpdated Nightly. (Either via interface or running firefox-56.0a1.en-US.win64.installer.exe [2017.07.02 mozilla central]).  This is in a Windows 8 machine!\n\n\nActual results:\n\nAfter restarting, nothing showed the bars were empty, the content was empty.  However (some?) buttons areas in the navigation bar were useful (tool tips show up).  A click above on the area below the close button made the options panel to show and I was able to restart in safe mode.  Firefox works in Safe Mode.  However, Firefox will not work with a new profile, even after refreshing Firefox.  I reverted to 2017.07.01. The 2017.07.03 does not fix anything for me.\n\n\nExpected results:\n\nFirefox Nightly should work mostly after an update.", "id": 12445163, "creation_time": "2017-07-04T00:08:07Z", "author": "johngraciliano@gmail.com"}, {"text": "\n\n*** This bug has been marked as a duplicate of bug 1377866 ***", "id": 12445279, "creation_time": "2017-07-04T02:36:27Z", "author": "yfdyh000@gmail.com"}, {"text": "John, if you get a chance could you attach your about:support? I will make sure the configuration is included in the fix for bug 1377866.", "id": 12445393, "creation_time": "2017-07-04T05:00:38Z", "author": "dvander@alliedmods.net"}], "attachments": []}
```

Comment:
Let's only add the `whiteboard` field (here and in the following), so the test is more readable